newsService/src/test/java/com/kovan/NewsServiceIT.java

```diff
@@ -14,7 +14,7 @@
 
 @ExtendWith(SpringExtension.class)
 @SpringBootTest(webEnvironment = RANDOM_PORT)
-public class NewsServiceIT {
+class NewsServiceIT {
```

Comment:
you have modified Test class so please attach screen shot for all testcases.

Reply:
Screenshot attached.